test/integration/pages_load_ok.playwright.test.js

```diff
@@ -52,7 +52,7 @@ test.describe('with a valid session, the user can access the later form pages',
     await startPage.navigateHere()
     await startPage.clickStartNow()
 
-    await checkRouteResponse(page, '/dataset', [200, 304])
+    await checkRouteResponse(page, '/check/dataset', [200, 304])
   })
 
   test('/geometry-type', async ({ page }) => {
```

Comment:
Missing update to the test descriptions (like in the tests above).

Looks OK, otherwise.

Reply:
```suggestion
  test('/check/geometry-type', async ({ page }) => {
```